src/lib/maplayers.js

```diff
@@ -1,33 +1,82 @@
 import L from 'leaflet'
-import TclCountriesBiomesPasData from '../tcl_countries_biomes_pas'
-
-const getTclCountriesBiomesPasLayer = () => {
-  return L.geoJSON(TclCountriesBiomesPasData, {
-    style: {
-      color: '#EB5424',
-      weight: 1,
-      strokeOpacity: 0.8,
-      fillColor: '#EB5424',
-      fillOpacity: 0.2,
-    },
-  }).bindPopup(layer => {
-    const name = layer.feature.properties.scl_name
-    const area = parseInt(layer.feature.properties.scl_total_area, 10)
-    const msg = `
-      <div>
-        Name: ${name}
-      </div>
-      <div>
-        Total Area: ${area}km<sup>2</sup>
-      </div>
-    `
-
-    return msg
-  })
-}
+import SecureTileLayer from './leaflet-secure-tilelayer'
+import { API_ROOT, TILE_API_ROOT } from '../api-config'
+import SecureApi from './api'
+import TokenUtil from './token'
+
+class MapLayers {
+  constructor() {
+    this.api = new SecureApi()
+    this.tokenUtil = new TokenUtil()
+    this.defaultTileLayerConfig = {
+      attribution: '&copy; WCS',
+      subdomains: ['tilea', 'tileb', 'tilec', 'tiled'],
+      token: this.tokenUtil.token,
+    }
+  }
+
+  get token() {
+    return this.tokenUtil.token
+  }
+
+  getTclLayer(country) {
+    const date = '2006-01-01'
```

Comment:
Could we maybe make date also a constructor parameter, and temporarily set both it and country as `let`s at the top of `leafletMap.index.js`? That way it will be easier to change them from other parts of the app down the road, when we have a date slider and country picker.